include/zephyr/net/ieee802154_radio.h

```diff
@@ -1241,6 +1241,11 @@ struct ieee802154_config {
 			 * in CPU byte order
 			 */
 			uint16_t short_addr;
+
+			/**
+			 * Flag for clearing enh ACK header IE (e.g during stack reset)
```

Comment:
Please improve the documentation, it's unclear what effect setting this flag has on the resulting configuration. For instance, is it possible to clear all IEs and set a new one in just one call to `configure`? It doesn't follow from the documentation and requires reading the implementation.

Personally, I'd also suggest to name it `purge` instead of `clear` to strongly imply that it removes all existing entries, but that's just a preference I guess.

Reply:
Extended documentation. Please take a look if it makes sense now